streams/src/commonMain/kotlin/com/mirego/trikot/streams/reactive/PublisherExtensions.kt

```diff
@@ -110,6 +111,10 @@ fun <T, R> Publisher<T>.filterNotNull(block: ((T) -> R?)): Publisher<R> {
     return this.filter { block(it) != null }.map { block(it)!! }
 }
 
+fun <T> Publisher<T>.timeout(duration: Duration, message: String = "Default timeout message"): Publisher<T> {
```

Comment:
Faudrait pouvoir passer un  timerFactory et defaulter à la FoundationConfiguration.timerFactory

Reply:
The processor has this configuration, its just the shorthand when you make your code (which I think will never specify the factory)